packages/www/components/cards/cards/category.tsx

```diff
@@ -6,15 +6,15 @@ type Props = {
   img: { src: string; alt?: string; sx?: SxStyleProp };
   link: { href: string; asPath?: string };
   title: string;
-  description: string;
+  // description: string;
```

Comment:
@wohlner rather than remove this field from the component we should mark it as optional. The syntax for this is `description?: string;`

Reply:
Unless you want to remove it from all cards.

Reply:
I don't know how to mark it optional. I just don't want the description to show up on the black cards on /docs. Can you please make the change to make description optional? @adamsoffer

Reply:
Fixed